Source/WebKit/UIProcess/API/APIInjectedBundleClient.h

```diff
@@ -44,6 +45,9 @@ class InjectedBundleClient {
     virtual void didReceiveMessageFromInjectedBundle(WebKit::WebProcessPool&, const WTF::String&, API::Object*) { }
     virtual void didReceiveSynchronousMessageFromInjectedBundle(WebKit::WebProcessPool&, const WTF::String&, API::Object*, CompletionHandler<void(RefPtr<API::Object>)>&& completionHandler) { completionHandler(nullptr); }
     virtual RefPtr<API::Object> getInjectedBundleInitializationUserData(WebKit::WebProcessPool&) { return nullptr; }
+#if (PLATFORM(GTK) || PLATFORM(WPE))
+    virtual RefPtr<API::Object> getInjectedBundleInitializationUserDataByPage(WebKit::WebProcessPool&, WebKit::WebPageProxy*) { return nullptr; }
```

Comment:
Why a raw pointer here? Shouldn't it be non-nullable? Use pass by reference unless you're sure you really need it to be nullable:

```suggestion
    virtual RefPtr<API::Object> getInjectedBundleInitializationUserDataByPage(WebKit::WebProcessPool&, WebKit::WebPageProxy&) { return nullptr; }
```